src/misc/support.cpp

```diff
@@ -208,6 +208,39 @@ FILE_unique_ptr make_fopen(const char *fname, const char *mode)
 	return f ? FILE_unique_ptr(f) : nullptr;
 }
 
+static constexpr int kb_divisor = 1024L;
+static constexpr int sector_size = 512L;
```

Comment:
Suggest moving these both inside their functions, as they're not being reused in a bunch of places.